lib/ProMotion/menu/drawer.rb

```diff
@@ -8,10 +8,19 @@ class Drawer < MMDrawerController
 
       def self.new(center=nil, options={})
         menu = alloc.init
+        menu.send(:screen_init) if menu.respond_to?(:screen_init)
```

Comment:
alternatively, could set the screen options instance variable to `{}`

Reply:
Wouldn't this happen twice now that you have that init method already doing this?

Reply:
Yeah, I noticed that as well. It's a strange pattern within ProMotion itself. I'll remove it.